src/Drive.php

```diff
@@ -258,18 +285,25 @@ public function createFolder(string $path): bool
         return true;
     }
 
+    /**
+     * @todo This function is not implemented yet! Place, name and signature of the function might change!
+     */
     public function getResourceMetadata(string $path = "/"): \stdClass
     {
-        throw new \Exception("This function is not implemented yet.");
+        throw new \Exception("This function is not implemented yet! Place, name and signature of the function might change!");
     }
 
+    /**
+     * @todo This function is not implemented yet! Place, name and signature of the function might change!
+     */
     public function getResourceMetadataById(string $id): \stdClass
     {
-        throw new \Exception("This function is not implemented yet.");
+        throw new \Exception("This function is not implemented yet! Place, name and signature of the function might change!");
     }
 
     /**
-     * update file content if the file already exists
+     * Update the file content
```

Comment:
this method creates new file  if file doesn't exist and updates content if the file exists